charts/castai-agent/templates/deployment.yaml

```diff
@@ -77,6 +77,18 @@ spec:
             - name: shared-metadata
               mountPath: /agent-metadata
           env:
+            - name: SELF_POD_NAME
```

Comment:
why do we need these?

Reply:
Updated to remove as we only need namespace